client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -297,6 +298,17 @@ class BusinessDetails extends Component {
 		} );
 	}
 
+	getSelectControlProps( getInputProps, name ) {
+		const { className, ...props } = getInputProps( name );
+		return {
+			...props,
+			className: classnames(
+				`woocommerce-profile-wizard_${ name }`,
```

Comment:
Should these be a bit more BEM style selectors?  For example:

`woocommerce-profile-wizard__selling-venues-control`

Reply:
Good idea! I added that change in commit 0327159.